src/main/java/edu/byu/cs/controller/SubmissionController.java

```diff
@@ -209,8 +215,7 @@ private static Submission getMostRecentSubmission(String netId, Phase phase) {
 
         Collection<String> currentlyGrading = TrafficController.sessions.keySet()
                 .stream()
```

Comment:
@pawlh Is this use of streams also potentially problematic like the issue we saw in #142?

Reply:
Ok. If it's an issue, it will be resolved in a different pull request anyways.